packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -617,6 +621,30 @@ export class AtomicSearchBox {
     );
   }
 
+  private getSuggestionBackgroundClass(
+    isSelected: boolean,
+    side: 'left' | 'right',
+    item: SearchBoxSuggestionElement
+  ) {
+    if (isSelected) {
+      return 'bg-neutral-light';
+    }
+    if (side === 'left' && item.linkedBy) {
```

Comment:
I haven't tested it, but is it currently possible to show instant results on the left and suggestions on the right?

My impression was that, currently, both sides are treated equally. I wouldn't mind deciding that one side is dependent on the other, since that may allow some accessibility improvements.

Reply:
>  but is it currently possible to show instant results on the left and suggestions on the right?

Not that I'm aware of.